File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V3/TableSchema.cs
using System.Collections.Generic;

namespace FactSet.Protobuf.Stach.Extensions.V3
{
    internal class TableSchema
    {
        public string Name;
        public string Version;
        public List<Dictionary<string, object>> Columns;
        public List<Dictionary<string, string>> Rows;
        public string ViewName;
        public Dictionary<string, string> ViewHeaders;
        public List<string> ViewColumns;
        public Dictionary<string, object> GroupResult;
        public Dictionary<string, object> SplitResult;
        public Dictionary<string, object> PrimaryKeys;
    }
}

File: python/src/fds/protobuf/stach/extensions/v3/TableSchema.py
class TableSchema:
    def __init__(self):
        self.Name = None
        self.Version = None
        self.Columns = []
        self.Rows = []
        self.ViewName = None
        self.ViewHeaders = {}
        self.ViewColumns = []
        self.GroupResult = {}
        self.SplitResult = {}
        self.PrimaryKeys = {}